jsapp/js/components/reports/reports.scss

```diff
@@ -198,27 +200,27 @@
   }
 
   &--vertical {
-    background-image: url("~../img/chart-vertical.png");
```

Comment:
Does [the tilde in the url()](https://stackoverflow.com/questions/39535760/what-does-a-tilde-in-a-css-url-do) do anything here? From my reading it doesn't, and can be removed, but I might be missing something.

Reply:
The tilde could possibly be useful if we use it to set an alias for the images folder. That way the "relative path" doesn't have to change when we change the .scss file (or the way it's being imported.)